.github/workflows/required-reviewers-release.yml

```diff
@@ -0,0 +1,68 @@
+name: Required release approval
+on:
+  pull_request_target:
+    types:
+      - opened
+
+      # This is triggered when the base branch changes; handles the case where you open a PR against one branch
+      # then change the base branch to a release branch.
+      - edited
```

Comment:
Is there a way to trigger this when the set of approvers has changed? Seems like we would want to re-run validation after a user approves, without needing to push changes to the branch. Or am I missing something?

Reply:
Great catch. We need to trigger this on pull_request_review as well.